release/artifact-install.pom

```diff
@@ -0,0 +1,469 @@
+                            <version>${project.version}</version>
+                            <packaging>jar</packaging>
+                            <file>common-${project.version}.jar</file>
+                            <generatePom>true</generatePom>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>install-connector</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>install-file</goal>
+                        </goals>
+                        <configuration>
+                            <groupId>jakarta.tck</groupId>
+                            <artifactId>connector</artifactId>
+                            <version>${project.version}</version>
+                            <packaging>jar</packaging>
+                            <file>connector-${project.version}.jar</file>
+                            <generatePom>true</generatePom>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>install-ejb30</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>install-file</goal>
+                        </goals>
+                        <configuration>
+                            <groupId>jakarta.tck</groupId>
+                            <artifactId>ejb30</artifactId>
+                            <version>${project.version}</version>
+                            <packaging>jar</packaging>
+                            <file>ejb30-${project.version}.jar</file>
+                            <generatePom>true</generatePom>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>install-ejb32</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>install-file</goal>
+                        </goals>
+                        <configuration>
+                            <groupId>jakarta.tck</groupId>
+                            <artifactId>ejb32</artifactId>
+                            <version>${project.version}</version>
+                            <packaging>jar</packaging>
+                            <file>ejb32-${project.version}.jar</file>
+                            <generatePom>true</generatePom>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>install-el-tck</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>install-file</goal>
+                        </goals>
+                        <configuration>
+                            <groupId>jakarta.tck</groupId>
+                            <artifactId>el-tck</artifactId>
```

Comment:
Should this be changed to "el-platform-tck" or rename el module artifact id to "el-tck"

Reply:
I think it would be better to have a common naming convention for all modules in platform TCK. Maybe better to keep  "el-tck" and rename the artifact id in module el.

Reply:
@alwin-joseph - Please note rename of EL standalone TCK, should not duplicate balloted EL standalone TCK artifact.That will impact future service releases of EL standalone TCK.

Reply:
For standalone el-tck/ module, we have below :
```
<artifactId>jakarta-expression-language-tck</artifactId>
<name>el-tck</name>
```

For platform el/ module
```
<artifactId>el-platform-tck</artifactId>
<name>el</name>
``` 
Maybe we can continue to use `el-platform-tck` as artifact for platform(so as to not confuse with el-tck module `name` of the standalone) and correct the same here in line 128, for now.